packages/turf-center-median/index.js

```diff
@@ -0,0 +1,71 @@
+import centerMean from '@turf/center-mean';
+import distance from '@turf/distance';
+import { point, isObject, isNumber } from '@turf/helpers';
+import { featureEach } from '@turf/meta';
+import { getCoord } from '@turf/invariant';
+
+/**
+ * Takes a {@link FeatureCollection} of points and calculates the median center,
+ * algorithimically. It uses the algorithm described in:
+ *
+ * Harold W. Kuhn and Robert E. Kuenne, “An Efficient Algorithm for the
+ * Numerical Solution of the Generalized Weber Problem in Spatial
+ * Economics,” _Journal of Regional Science_ 4, no. 2 (1962): 21–33,
+ * doi:{@link https://doi.org/10.1111/j.1467-9787.1962.tb00902.x}.
+ *
+ * James E. Burt, Gerald M. Barber, and David L. Rigby, _Elementary
+ * Statistics for Geographers_, 3rd ed., New York: The Guilford
+ * Press, 2009, 150–151.
+ *
+ * @name centerMedian
+ * @param {FeatureCollection<Point>} features GeoJSON Feature or Geometry
+ * @param {Object} [options={}] Optional parameters
+ * @param {string} [options.weight] the property name used to weight the center
+ * @param {number} [options.tolerance=0.001] the difference in distance between candidate medians at which point the algorighim stops iterating.
+ * @returns {Feature<Point>} The median center of the collection
+ * @example
+ * var points = turf.featureCollection([turf.point([0, 0], turf.point([1, 0]), turf.point([0, 1]), turf.point([5, 8])]);
```

Comment:
We've got a FANCY ⚡️ new module called `points()` which allows you to do the following:

```js
cons points = turf.points([
  [0, 0],
  [1, 0],
  [0, 1],
  [1, 1]
]);